addons/sms/tools/sms_api.py

```diff
@@ -44,7 +44,11 @@ def _send_sms_batch(self, messages, delivery_reports_url=False):
               }, ...
           ]```
         """
-        return self._contact_iap('/iap/sms/3/send', {'messages': messages, 'webhook_url': delivery_reports_url})
+        return self._contact_iap('/iap/sms/3/send', {
```

Comment:
While we're at it, we could move this to the new API route name, `/api/sms/3/send` (only the name changes, the route is the same).